package.json

```diff
@@ -2,10 +2,11 @@
   "name": "optic-release-automation-action",
   "version": "2.3.0",
   "description": "Automate the release process of your npm modules by providing them OTP on-demand.",
-  "main": "index.js",
+  "main": "action.js",
```

Comment:
we really need to restructure this repo moving all the source code to a src folder as we did in the merge action. do you mind taking the chance to do so in this PR please?

Reply:
I thought the same, but I was afraid to add confusion for the reviewers

Reply:
can you create an issue for this? we'll do it on the next round, I agree it would be hard to spot changes if we did it in this PR